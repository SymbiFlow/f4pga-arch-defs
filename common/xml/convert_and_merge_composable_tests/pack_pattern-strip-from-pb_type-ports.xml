<?xml version="1.0"?>
<!--
  Strip pack_pattern's from input/output tags on pb_types.

  This allows us to add the pack_pattern annotation at the leaf values which
  don't have an interconnect. It makes it explicit which ports are involved in
  the chain (and hence won't end up on fabric).

  -->
<xml>
 <pb_type>
  <input name="in1">
    <pack_pattern />
  </input>
  <input name="in2" />
  <output name="o1" />
  <output name="o2">
   <pack_pattern>
    <innertag />
   </pack_pattern>
  </output>
 </pb_type>
</xml>
