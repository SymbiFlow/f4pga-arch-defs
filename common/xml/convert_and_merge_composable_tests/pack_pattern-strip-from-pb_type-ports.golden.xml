<?xml version="1.0"?>
<xml>
  <pb_type>
    <input name="in1"/>
    <input name="in2"/>
    <output name="o1"/>
    <output name="o2"/>
  </pb_type>
</xml>
